lib/baw-workers/exceptions.rb

```diff
@@ -10,5 +10,6 @@ class HarvesterAudioToolError < HarvesterIOError; end
     class HarvesterEndpointError < HarvesterError; end
     class HarvesterAnalysisError < HarvesterError; end
     class AnalysisCacheError < StandardError; end
+    class PartialPayloadMissing < StandardError; end
```

Comment:
This also appears in https://github.com/QutBioacoustics/baw-workers/pull/42/files#diff-afb1ab8a76efa179c9d7f74ea6e0d2bcR3